src/Core/Echo.ControlFlow/Serialization/Dot/DefaultInstructionFormatter.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Echo.ControlFlow.Serialization.Dot
+{
+    internal sealed class DefaultInstructionFormatter<TInstruction> : IInstructionFormatter<TInstruction>
```

Comment:
Since this is marked `internal` and probably will never have properties defined, perhaps make this a singleton?